cdk-infra/lib/constructs/worker/worker-construct.ts

```diff
@@ -84,7 +84,7 @@ export class WorkerConstruct extends Construct {
 
     const taskDefLogGroup = new LogGroup(this, 'workerLogGroup');
     const taskDefinition = new FargateTaskDefinition(this, 'workerTaskDefinition', {
-      cpu: 2048,
+      cpu: 4096,
```

Comment:
increased the CPU amount because it appeared that at 2 vCPUs (which is what a `cpu` value of 2048 represents) that the tasks were maxing out their CPU usage.